guides/using_hugging_face_integrations.md

```diff
@@ -0,0 +1,158 @@
+
+```python
+import gradio as gr
+
+iface = gr.Interface.load("huggingface/Helsinki-NLP/opus-mt-en-es",
+  examples=[["Hello! My name is Omar"]]
+)
+
+iface.launch()
+```
+
+Let's go over some of the key differences:
+
+* `Interface.load()` is used instead of the usual `Interface()`.
+* `Interface.load()` receives a string with the prefix `huggingface/`, and then the model repository ID.
+* Since the input, output and prediction functions are not needed, you only need to modify the UI parts (such as `title`, `description`, and `examples`).
+* There is no need to install any dependencies (except Gradio) since you are not loading the model on your computer.
+
+You might notice that the first inference takes about 20 seconds. This happens since the Inference API is loading the model in the server. You get some benefits afterward:
+
+* The inference will be much faster.
+* The server caches your requests.
+* You get built-in automatic scaling.
+
+## Hosting your Gradio demos
+
+
+[Hugging Face Spaces](hf.co/spaces) allows anyone to host their Gradio demos freely. The community shares oven 2,000 Spaces. Uploading your Gradio demos take a couple of minutes. You can head to [hf.co/new-space](https://huggingface.co/new-space), select the Gradio SDK, create an `app.py` file, and voila! You have a demo you can share with anyone else.
+
+## Building demos based on other demos
+
+You can use the existing Spaces to tweak the UI or combine multiple demos. Let's find how to do this! First, let's take a look at an existing demo that does background removal. 
+
+This is a Gradio demo [already shared](https://huggingface.co/spaces/eugenesiow/remove-bg) by a community member. You can load an existing demo using `Interface` in a syntax similar to how it's done for the Inference API. It just takes two lines of code and with a prefix `spaces`.
+
+```python
+import gradio as gr
+
+gr.Interface.load("spaces/eugenesiow/remove-bg").launch()
+```
+
+The code snippet above will load the same interface as the corresponding Space demo.
+
+<iframe src="https://hf.space/gradioiframe/eugenesiow/remove-bg/+" frameBorder="0" height="450" title="Gradio app" class="container p-0 flex-grow space-iframe" allow="accelerometer; ambient-light-sensor; autoplay; battery; camera; document-domain; encrypted-media; fullscreen; geolocation; gyroscope; layout-animations; legacy-image-formats; magnetometer; microphone; midi; oversized-images; payment; picture-in-picture; publickey-credentials-get; sync-xhr; usb; vr ; wake-lock; xr-spatial-tracking" sandbox="allow-forms allow-modals allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts allow-downloads"></iframe>
```

Comment:
note that you can change the height parameter in <iframe> to avoid the vertical scroll bars and the interface getting cut at the bottom. For example, here height="920" is the perfect number

Reply:
But this will behave differently on mobile, since the interface becomes a vertical layout.

Reply:
Good idea, modified it!